integration-test/import-control.xml

```diff
@@ -25,8 +25,10 @@
   <allow pkg="java.util"/>
   <allow pkg="java.sql"/>
   <allow pkg="java.io"/>
+  <allow pkg="java.text"/>
   <allow pkg="org\.junit.*" regex="true"/>
   <allow pkg="org\.apache\.iotdb\.itbase" regex="true"/>
   <allow pkg="org\.apache\.iotdb\.it.*" regex="true"/>
+  <allow pkg="org\.apache\.iotdb\.db\.it\.utils\.TestUtils.*" regex="true"/>
```

Comment:
If you want to add another file to import whitelist, please change `integration-test/checkstyle.xml` like this:
![image](https://user-images.githubusercontent.com/3821212/176343032-34745aa5-701b-4969-87a2-9769c0c32793.png)

Change this line to `<property name="path" value="^.*(IT|TestUtils|TestConstant)\.java$"/>`

Reply:
Got it.